float32[] x_center
float32[] y_center
float32[] x_min
float32[] x_max
float32[] y_min
float32[] y_max
float32[] radius
